pkg/cmd/step/step_split_monorepo.go

```diff
@@ -206,9 +206,10 @@ func (o *StepSplitMonorepoOptions) Run() error {
 							return err
 						}
 					}
+					//ToDo: Why are we ignoring errors?
```

Comment:
No idea. My best guess would be that the comment implies that for example _.gitignore_ might already be added to version control and when calling `git add` on it you would get an error. However, this is not the case. You can call `git add` as often as you want. So if this call would return an error it would be an error in the whole logic/execution of executing the actual `git add`. So imo I don't think the error should be ignored here. 

Also, `split monorepo` is another one of these commands which I believe have no use. In fact, it is in the list of deprecated commands (_deprecated.go_). For all, I care we can remove it. Not sure whether we should do it as part of this pull request though. @abayer thoughts? 
If we keep it, I would handle the error and remove the comment.